### PR TITLE
DRAFT PoC of getting close to 99%

### PR DESCRIPTION
**Pull request by aryx**

### CODE REVIEW

**Comment by aryx** [comment]
@colleend this is just FYI. It is not meant to be merged. I need to split it in different clean PRs for fwcd.
I also need to wait for your recent PR to be accepted so I can rebase on it (no worry, no hurry).

File: grammar.js [hunk 1]
```diff
@@ -152,6 +159,7 @@ module.exports = grammar({
     top_level_object: $ => seq($._declaration, optional($._semis)),
 
     type_alias: $ => seq(
+      optional($.modifiers),
```
Comment (aryx) [inline grammar.js:162]: @colleend that's the thing I mentioned in your PR to fwcd that you could also include (small change).
Better to not have too many PRs, so for small changes  like that I think it's ok we pack multiple small changes in only 1 PR.

File: grammar.js [hunk 2]
```diff
@@ -614,7 +626,9 @@ module.exports = grammar({
 
     elvis_expression: $ => prec.left(PREC.ELVIS, seq($._expression, "?:", $._expression)),
 
-    check_expression: $ => prec.left(PREC.CHECK, seq($._expression, choice($._in_operator, $._is_operator), $._expression)),
+    check_expression: $ => prec.left(PREC.CHECK, seq($._expression, choice(
+									   seq($._in_operator, $._expression),
+									   seq($._is_operator, $._type)))),
```
Comment (aryx) [inline grammar.js:631]: @colleend that's the other fix to is_operator I mentioned in your PR to fwcd.

File: grammar.js (hunk 1 above)
Comment (colleend) [reply grammar.js:162]: I see, good find! I can put this fix in my PR then!

File: grammar.js (hunk 2 above)
Comment (colleend) [reply grammar.js:631]: oh awesome, I'll also include this one then!

**Comment by colleend** [comment]
should we close this draft out?

**Comment by aryx** [comment]
Yes!

**Pull request closed without merging**